src/main/java/com/xpanxion/java/assignments/student1/Worker1.java

```diff
@@ -1,25 +1,35 @@
 package com.xpanxion.java.assignments.student1;
 
 import com.xpanxion.java.assignments.DataAccess;
+import com.xpanxion.java.assignments.model.Department;
 
+import java.util.Map;
 import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Function;
+import java.util.stream.Collectors;
 
 public class Worker1 {
 
     public void ex1() {
         var products = DataAccess.getProducts();
-        var departments = DataAccess.getDepartments();
-        products
-                .stream()
-                .map(product -> product)
-                .forEach(p -> {
-                    departments.forEach(d -> {
-                        if (d.getId() == p.getDepartmentId()) {
-                            p.setDepartmentName(d.getName());
-                        }
-                    });
-                });
+        Map<Integer, Department> departmentMap = DataAccess.getDepartments().stream()
```

Comment:
So, I think we are getting ahead of ourselves here.  PRs have to be focused.  The title suggests that this PR is for ex 13, but it contains changes for 1, 2, and 3.  To keep my sanity, I will close this PR.  Can you update the other PRs?